main/main.cpp

```diff
@@ -581,12 +581,11 @@ Error Main::setup(const char *execpath, int argc, char *argv[], bool p_second_ph
 	GLOBAL_DEF("memory/multithread/thread_rid_pool_prealloc", 60);
 
 	GLOBAL_DEF("network/debug/max_remote_stdout_chars_per_second", 2048);
-	GLOBAL_DEF("network/debug/remote_port", 6007);
```

Comment:
Oh my bad, I hadn't noticed there were both the project and editor settings involved here. I'm fine with having the setting only at the editor level; if a need arises to have it project-specific, it can likely be added at that time.